src/main/java/decodes/hdb/HdbSiteDAO.java

```diff
@@ -519,4 +521,83 @@ else if (s2 == null || s2.length() == 0)
 			return false;
 		return s1.equals(s2);
 	}
+
+	@Override
+	public void fillCache()
```

Comment:
How likely is this to impact/help #255 ?

Reply:
Very, which is why I wasn't planning to merge this in until what you were working on was submitted and merged in.

Unless you think it would make that work easier?
I was expecting to rebase and adjust for whatever changes you need to make.

Reply:
FYI, I'd like to get this in sooner rather than later.